templates/vhost/_ssl.erb

```diff
@@ -2,8 +2,10 @@
 
   ## SSL directives
   SSLEngine on
+  <%- unless @mdomain -%>
   SSLCertificateFile      "<%= @ssl_cert %>"
   SSLCertificateKeyFile   "<%= @ssl_key %>"
+  <%- end -%>
   <%- if @ssl_chain -%>
```

Comment:
is this still useful? or should we add that to be excluded?

Reply:
According to [mod_ssl doc](https://httpd.apache.org/docs/current/en/mod/mod_ssl.html#sslcertificatechainfile):

> SSLCertificateChainFile became obsolete with version 2.4.8, when SSLCertificateFile was extended to also load intermediate CA certificates from the server certificate file.

Some supported old distributions might still require it (It looks like RedHat 6 has Apache 2.2)…